data/model/similar_user_model.py

```diff
@@ -0,0 +1,9 @@
+import pydantic
+from typing import List
+
+class SimilarUserRecord(pydantic.BaseModel):
+    user_id: int
+    similarity_score: float
```

Comment:
what is the range of this float?

Reply:
Not sure, the docs for implicit weren't very clear.